scene/resources/visual_shader.cpp

```diff
@@ -949,9 +949,6 @@ Vector2 VisualShader::get_node_position(Type p_type, int p_id) const {
 Ref<VisualShaderNode> VisualShader::get_node(Type p_type, int p_id) const {
 	ERR_FAIL_INDEX_V(p_type, TYPE_MAX, Ref<VisualShaderNode>());
 	const Graph *g = &graph[p_type];
-	if (!g->nodes.has(p_id)) {
-		return Ref<VisualShaderNode>();
-	}
 	ERR_FAIL_COND_V(!g->nodes.has(p_id), Ref<VisualShaderNode>());
```

Comment:
@Mickeon This line is expanded to this https://github.com/godotengine/godot/blob/446e7a7c228d19341392ec3b02417244fe7dfe03/core/error/error_macros.h#L439

Reply:
I understood. I do believe it would be a good idea to open a separate PR for it, too, as unlike the comments it's pretty clear-cut.

Reply:
Okay no problem, that can be done, of course, I will separate these changes from the comments. Thanks.